test/mkmf/test_configuration.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: false
+require_relative 'base'
+
+class TestMkmfConfiguration < TestMkmf
+  def test_verbose_arg
+    assert_separately([], %w[- -- --with-verbose], <<-'end;') #do
+      assert_equal(true, with_config("verbose"))
+    end;
+  end
+
+  def test_verbose_with_rbconfig_disabled
```

Comment:
I want to change the method name `test_verbose_with_rbconfig_disabled` to `test_verbose_with_rbconfig_verbose_disabled`, and `test_verbose_with_rbconfig_enabled` to `test_verbose_with_rbconfig_verbose_enabled`.

Reply:
Renamed the method names.